ci/env-create.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
```

Comment:
Are we sure this file is well-formed?

I was looking into the docs https://docs.gitlab.com/ee/ci/pipelines/job_artifacts.html#artifactsreportsdotenv.
> The .env file can’t have empty lines or comments (starting with #).